ConfigSpace/hyperparameters/normal_integer.pyx

```diff
@@ -195,7 +195,9 @@ cdef class NormalIntegerHyperparameter(IntegerHyperparameter):
                                             q=self.q, log=self.log, meta=self.meta)
 
     def is_legal(self, value: int) -> bool:
-        return isinstance(value, (int, np.int32, np.int64))
+        return (isinstance(value, (int, np.int32, np.int64))) and \
```

Comment:
Likewise for here `isinstance(value, (int, np.integer))`

Reply:
Done.